src/card.css

```
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 410px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__product {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  height: 36px;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-size: 10px;
+  height: 14px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
```

```suggestion
  margin-bottom: 16px;
  height: 14px;
  font-size: 10px;
  line-height: 14px;
  color: #616070;
```
remember about property order, also check in other places